### DISCUSSION TITLE
Apple M1 & M2

### DISCUSSION THREAD

**Discussion by dtigue** [esnet/iperf#1387] [General]
Just had a quick question I thought might could be answered here. Does anyone know if there are any plans to port this to the Apple M1 or M2 systems? If not, will the intel based version work correctly on the m1 and m2 Macs? Thanks!

**Comment by bmah888** [comment]
ESnet only distributes source code for iperf3. We neither distribute nor support executables that are compiled for a particular hardware/software platform.

iperf3 doesn't target any particular CPU (in other words it's not written specifically for Intel or Apple Silicon). As far as I'm aware, if you compile it on an Apple Silicon Mac, it should work correctly. An Intel binary might work under Rosetta 2, but it might not characterize performance accurately.
